step.sh

```diff
@@ -9,7 +9,7 @@ magenta=$'\e[35m'
 cyan=$'\e[36m'
 reset=$'\e[0m'
 
-MERGES=$(git log $(git merge-base --octopus $(git log -1 --merges --pretty=format:%P))..$(git log -1 --merges --pretty=format:%H) --pretty=format:%s)
+MERGES=$(git log --pretty=format:%s $(git merge-base --octopus $(git log -1 --merges --pretty=format:%P))..$(git log -1 --merges --pretty=format:%H))
```

Comment:
`pretty` w subshellach i w L21 też nie powinno iść przed `--merges` i `-1`?

Reply:
Jeżeli dobrze rozumiem komunikat błędu `fatal: option '--pretty=format:%s' must come before non-option arguments` to w subshellach i w linijce 21 jest dobrze bo  przed `pretty` nie ma żadnego non-option argumentu. Teraz jeszcze się zastanawiam czy te zmiany mają sens wgl bo jak teraz testuje u siebie lokalnie w cmd obie komendy działają niezależnie gdzie znajduje się `pretty`, a na bitrise wywala ten bład powyżej. Chyba jeszcze to troche potestuje i poczytam na ten temat. Myślisz, że może to być wina np stacku na bitrise ?

Reply:
Teraz na bitrise jest teraz git 2.40
https://github.com/bitrise-io/bitrise.io/blob/master/system_reports/MACOS/M1/osx-xcode-14.3.x-ventura.log#L54
U mnie lokalnie na tej samej wersji też działa `--pretty` nieważne gdzie jest.